log/logger.go

```diff
@@ -0,0 +1,26 @@
+package log
+
+import "github.com/aquasecurity/postee/log/stdoutlogger"
+
+var Logger LoggerType = initDefaultLogger()
+
+type LoggerType interface {
+	Info(args ...interface{})
```

Comment:
method to set minimal logging level is missed
like `SetLogLevel("DEBUG")` to write debug messages to log. Or at least flags to enable debug or disable logs at all. Like in trivy

Reply:
Done